pkg/os/util_linux.go

```diff
@@ -10,8 +10,22 @@ import (
 )
 
 func WriteToFileAsRoot(reason, content, filepath string) error {
+	return writeToFileAsRoot(reason, content, filepath, false)
+}
+
+func AppendToFileAsRoot(reason, content, filepath string) error {
+	return writeToFileAsRoot(reason, content, filepath, true)
+}
+
+func writeToFileAsRoot(reason, content, filepath string, append bool) error {
 	logging.Infof("Will use root access: %s", reason)
-	cmd := exec.Command("sudo", "tee", filepath) // #nosec G204
+	var append_option string
+	if append {
+		append_option = "-a"
+	} else {
+		append_option = ""
```

Comment:
I think you don't need this, uninitialized string is probably already "" already in go.